resources/views/forum/topics/_header_counter.blade.php

```diff
@@ -34,3 +34,22 @@
     <div class="counter-box__line u-forum--bg">
     </div>
 </div>
+@if(priv_check('ForumTopicModerate', $topic)->can())
+    <div class="counter-box counter-box--forum">
+        <div class="counter-box__content">
+            <div class="counter-box__title">
+                {{ trans('forum.topics.show.deleted-posts') }}
+            </div>
+
+            <div class="counter-box__count js-forum__deleted-count">
+                @if ($newTopic)
```

Comment:
not sure why I used separate span for the total. Can you change it as well? Also inlining it (`{{ $newTopic ? 0 : $... }}`) probably isn't too long